httpony/server.py

```diff
@@ -9,9 +9,16 @@
 
 
 def main():
+    """Read arguments if given"""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-l", "--listen", help="Set ip/hostname for HTTPony to use",
+                        default="localhost")
+    parser.add_argument("-p", "--port", help="Set port for HTTPony to use",
```

Comment:
`run_simple` wants an integer port. You can set the `type=int` and use an integer instead of a string default on `add_argument`. Doing this will add a validation so that a user can't provide `--port=lolz`.